Rakefile

```diff
@@ -3,6 +3,13 @@ require "bundler"
 
 Bundler.setup
 
+module TempFixForRakeLastComment
+  def last_comment
+    last_description
+  end 
+end
+Rake::Application.send :include, TempFixForRakeLastComment
```

Comment:
"temp"

Reply:
Nothing is as permanent as a temporary fix :D